tests/js/cursor/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -316,12 +319,15 @@ parsers: ["babel", "typescript", "flow"]
 printWidth: 80
                                                                                 | printWidth
```

Comment:
The printWidth indicator needs to be shifted 6 characters to the right for these tests.

Reply:
`.match(/^\s+1 \| /)[0].length` Big file more than `6`, I'll fix it.